tests/test_ops.py

```diff
@@ -0,0 +1,121 @@
+"""
+Unit tests for GPI2 operation and its utility functions.
+"""
+import pytest
+import torch
+import jax
+
+import pennylane as qml
+from pennylane import numpy as np
+import pennylane.math as math
+import tensorflow as tf
+from ionizer.ops import GPI
+
+jax.config.update("jax_enable_x64", True)
+
+interfaces = [
+        "autograd",
+        "torch",
+        "tf",
+        "jax",
+        "auto",
+        None
+    ]
+two_pi = 2 * np.pi
+
+def get_GPI_matrix(phi):
+    return np.array([[0, math.exp(-1j * phi)], [math.exp(1j * phi), 0]])
+
+
+class TestGPI:
+    @staticmethod
+    def circuit(phi): 
+        qml.Hadamard(wires=0)
+        GPI(phi, wires=0)
+        return qml.expval(qml.PauliZ(wires=0))
+    
+    @staticmethod
+    def circuit_unitary(phi): 
+        qml.Hadamard(wires=0)
+        gpi_matrix = get_GPI_matrix(phi)
+        qml.QubitUnitary(gpi_matrix, wires=0)
+        return qml.expval(qml.PauliZ(wires=0))
+    
+    @staticmethod
+    def interface_array(x, interface):
+        match interface:
+            case None:
+                return x
+            case "jax":
+                jax.config.update("jax_enable_x64", True)
+                return jax.numpy.array(x)
+            case "torch":
+                return torch.tensor(x)
+            case "tf":
+                return tf.Variable(x)
+            case _:
+                return qml.numpy.array(x)
+            
+    @pytest.mark.parametrize("interface", interfaces)
+    def test_GPI_compute_matrix(self, interface):
+        phi_rand = np.random.rand() * two_pi
```

Comment:
Instead of `rand` can just pick a couple arbitrary values and use a `parametrize` decorator.

Reply:
Should be fixed